src/docs/development/platform-integration/add-to-app-android/add-flutter-screen.md

```diff
@@ -0,0 +1,300 @@
+---
```

Comment:
Didn't we already merge this page in a different PR?

Reply:
I rebased it away.